<launch>
    <node pkg="calib_image_saver" type="singleImageSaver" name="saver" output="screen">
        <remap from="/image_input" to="/stereo/left/image_raw"/>
        <param name="image_path" type="string" value="/home/px/ws/devel/lib/camera_model/blue93"/>
        <param name="image_name" type="string" value="IMG_"/>
        <param name="board_width" type="int" value="9"/>
        <param name="board_height" type="int" value="8"/>
        <param name="is_use_OpenCV" type="bool" value="false"/>
        <param name="is_show" type="bool" value="true"/>
    </node>

</launch>

